bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/inst/ProbeArmGeometry.scala

```diff
@@ -100,4 +124,10 @@ object ProbeArmGeometry {
     def toCanonicalArcsec: Point2D =
       new Point2D.Double(p.getX.toCanonicalArcsec, p.getY.toCanonicalArcsec)
   }
+
+  def guideStarPoint(ctx: ObsContext, guideStarCoords: Coordinates): Point2D = {
+    val baseCoords      = ctx.getBaseCoordinates.toNewModel
+    val guideStarOffset = Coordinates.difference(baseCoords, guideStarCoords).offset
+    new Point2D.Double(-guideStarOffset.p.toArcsecs, -guideStarOffset.q.toArcsecs).toCanonicalArcsec
```

Comment:
Unfortunately you'll need to rebase and resolve conflicts and/or errors here.  Something like

````scala
import edu.gemini.spModel.core.AngleSyntax._
new Point2D.Double(-guideStarOffset.p.arcsecs, -guideStarOffset.q.arcsecs).toCanonicalArcsec
````